core/src/main/java/io/snappydata/impl/SnappyHiveCatalog.java

```diff
@@ -222,11 +237,11 @@ public String catalogSchemaName() {
   }
 
   @Override
-  public void stop() {
+  public void close() {
     HMSQuery q = getHMSQuery();
     q.resetValues(HMSQuery.CLOSE_HMC, null, null, true);
     try {
-      this.hmsQueriesExecutorService.submit(q).get();
+      this.hmsQueriesExecutorService.submit(q).get(5, TimeUnit.SECONDS);
```

Comment:
Is it ok to ignore exception here as it might timeout now?

Reply:
This is during shutdown so any exceptions are ignored. Timeout should be fine because node is stopping in any case. Primarily to avoid blocking shutdown here for long (and sometimes it is the Cancellation that triggers any blocked operation to give up).